packages/@aws-cdk/aws-glue/test/test.table.ts

```diff
@@ -221,6 +222,393 @@ export = {
+                  "Fn::Join": [
+                    "",
+                    [
+                      "s3://",
+                      {
+                        Ref: "DatabaseBucket318AF64F"
+                      },
+                      "database"
+                    ]
+                  ]
+                },
+                Name: "database"
+              }
+            }
+          },
+          TableBucketDA42407C: {
+            Type: "AWS::S3::Bucket",
+            Properties: {
+              BucketEncryption: {
+                ServerSideEncryptionConfiguration: [
+                  {
+                    ServerSideEncryptionByDefault: {
+                      SSEAlgorithm: "aws:kms"
+                    }
+                  }
+                ]
+              }
+            },
+            DeletionPolicy: "Retain"
+          },
+          Table4C2D914F: {
+            Type: "AWS::Glue::Table",
+            Properties: {
+              CatalogId: {
+                Ref: "AWS::AccountId"
+              },
+              DatabaseName: {
+                Ref: "DatabaseB269D8BB"
+              },
+              TableInput: {
+                Description: "table generated by CDK",
+                Name: "table",
+                StorageDescriptor: {
+                  Columns: [
+                    {
+                      Name: "col",
+                      Type: "string"
+                    }
+                  ],
+                  Compressed: false,
+                  InputFormat: "org.apache.hadoop.mapred.TextInputFormat",
+                  Location: {
+                    "Fn::Join": [
+                      "",
+                      [
+                        "s3://",
+                        {
+                          Ref: "TableBucketDA42407C"
+                        },
+                        "/data/"
+                      ]
+                    ]
+                  },
+                  OutputFormat: "org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat",
+                  SerdeInfo: {
+                    SerializationLibrary: "org.openx.data.jsonserde.JsonSerDe"
+                  },
+                  StoredAsSubDirectories: false
+                },
+                TableType: "EXTERNAL_TABLE"
+              }
+            }
+          }
+        }
+      });
+
+      test.done();
+    },
+
+    'explicit SSE-KMS key'(test: Test) {
+      const stack = new cdk.Stack();
+      const database = new glue.Database(stack, 'Database', {
+        databaseName: 'database'
+      });
+      const encryptionKey = new kms.EncryptionKey(stack, 'MyKey');
+
+      new glue.Table(stack, 'Table', {
+        database,
+        tableName: 'table',
+        columns: [{
+          name: 'col',
+          type: Schema.string
+        }],
+        encryption: glue.TableEncryption.SSE_KMS,
+        encryptionKey,
+        storageType: glue.StorageType.Json,
+      });
+
+      expect(stack).toMatch({
```

Comment:
As a practice, it's better to use `to(haveResource(` and make more specific assertions. Otherwise, your tests are a bit too sensitive to changes.